stable/keycloak/templates/statefulset.yaml

```diff
@@ -107,7 +107,7 @@ spec:
             timeoutSeconds: {{ .Values.keycloak.livenessProbe.timeoutSeconds }}
           readinessProbe:
             httpGet:
-              path: /{{ .Values.keycloak.basepath }}/realms/master
+              path: {{ if ne .Values.keycloak.basepath "" }}/{{ .Values.keycloak.basepath }}{{ end }}/realms/master
```

Comment:
Could you fix the liveness probe as well?

Reply:
yes I can do that